### ISSUE TITLE
Where are the JavaScript communities and enthusiasts?

### ISSUE THREAD

**Issue by kytrinyx** [exercism/javascript#547]
As we move towards the launch of the new version of Exercism we are going to be ramping up on actively recruiting people to help provide feedback.

Our goal is to get to 100%: everyone who submits a solution and wants feedback should get feedback. Good feedback. You can read more about this aspect of the new site here: http://mentoring.exercism.io/

To do this, we're going to need a lot more information about where we can find language enthusiasts.

* Is JavaScript supported by one or more large organizations?
* Does JavaScript have an official community manager?
* Do you know of specific communities (online or offline) that are enthusiastic about JavaScript? (Chat communities, forums, meetups, student clubs, etc)
* Are there popular conferences for JavaScript? (If so, what are some examples?)
* Are there any organizations who are targeted specifically at getting certain subgroups or demographics interested in JavaScript? (e.g. kids, teenagers, career changers, people belonging to various groups that are typically underrepresented in tech?)
* Are there specific groups or programs dedicated to mentoring people in JavaScript?
* Are there popular newsletters for JavaScript?
* Is JavaScript taught at programming bootcamps? (If so, what are some examples?)
* Is JavaScript taught at universities? (If so, what are some examples?)

In other words: where do people care a lot and/or know a lot about JavaScript?

This is part of the project being tracked in https://github.com/exercism/meta/issues/103

**Comment by matthewmorgan** [comment]
Closing as wontfix per #556.
